Strings/Ukrainian (uk-UA)/Resources.resx

```diff
@@ -0,0 +1,2228 @@
+    </data>
+    <data name="C_Reader_SearchEngine.Title" xml:space="preserve">
+        <value>Пошукова система</value>
+        <comment>Search engine</comment>
+    </data>
+    <data name="C_Reader_StdMrgn.Description" xml:space="preserve">
+        <value>Перевизначає будь-які поля, установлені в книзі, щоб забезпечити узгоджений макет у читалці</value>
+        <comment>Overrides any margin specified in the book to provide consistent layout in the reader</comment>
+    </data>
+    <data name="C_Reader_StdMrgn.Title" xml:space="preserve">
+        <value>Стандартизація полів</value>
+        <comment>Standardize margins</comment>
+    </data>
+    <data name="C_Reader_TextAlign.Description" xml:space="preserve">
+        <value>Перевизначає будь-які параметри вирівнювання тексту, установлені в книзі, щоб забезпечити узгоджений макет у читалці</value>
+        <comment>Overrides any text alignment preference set in the book to provide consistent layout in the reader</comment>
+    </data>
+    <data name="C_Reader_TextAlign.Title" xml:space="preserve">
+        <value>Перевизначення вирівнювання тексту</value>
+        <comment>Override text alignment</comment>
+    </data>
+    <data name="C_ShowFavs.Content" xml:space="preserve">
+        <value>Показувати тільки обране</value>
+        <comment>Show only favorites</comment>
+    </data>
+    <data name="CB_Category.ToolTipService.ToolTip" xml:space="preserve">
+        <value>Категорії</value>
+        <comment>Categories</comment>
+    </data>
+    <data name="EBI_Author.Text" xml:space="preserve">
+        <value>Автор(и): </value>
+        <comment>Author(s) : </comment>
+    </data>
+    <data name="EBI_Cover1.Content" xml:space="preserve">
+        <value>Обрати обкладинку</value>
+        <comment>Select cover</comment>
+    </data>
+    <data name="EBI_Cover2.ToolTipService.ToolTip" xml:space="preserve">
+        <value>Повернути оригінальну обкладинку</value>
+        <comment>Restore original cover</comment>
+    </data>
+    <data name="EBI_Description.Text" xml:space="preserve">
+        <value>Опис: </value>
+        <comment>Description : </comment>
+    </data>
+    <data name="EBI_ErrMsg1" xml:space="preserve">
+        <value>Номер в серії не може бути порожнім</value>
+        <comment>Series number cannot be empty</comment>
+    </data>
+    <data name="EBI_ErrMsg2" xml:space="preserve">
+        <value>Номер в серії некоректний</value>
+        <comment>Series number not valid</comment>
+    </data>
+    <data name="EBI_Genre.Text" xml:space="preserve">
+        <value>Жанр(и): </value>
+        <comment>Genre(s) : </comment>
+    </data>
+    <data name="EBI_Language.Text" xml:space="preserve">
+        <value>Мова(и): </value>
+        <comment>Language(s) : </comment>
+    </data>
+    <data name="EBI_Msg1.Text" xml:space="preserve">
+        <value> • (множина) - Використовуйте крапку з комою ( ; ) щоб відокремити декілька елементів.</value>
```

Comment:
There is no universal "s" in Ukrainian to mark plurals like in English. Plurals can have many different endings in different words and cases: "и", "і", "ї", "а" etc. This is why I wrote "(plurals)" here instead of every possible ending or just random one of them.